### PR TITLE
create feedback-slider

### PR DESCRIPTION
**Pull request by Abdybakieva**

### CODE REVIEW

File: src/components/feedback-slider/FeedbackSlider.jsx [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+   },
+   '& .slick-prev': {
+      position: 'absolute',
+      top: '289px',
+      zIndex: '8',
+      left: '273px',
+   },
+   '& .slick-next:hover, .slick-prev:hover': {
+      circle: {
+         fill: 'url(#paint0_linear_92_5157)',
```
Comment (gulli301185) [inline src/components/feedback-slider/FeedbackSlider.jsx:178]: checked)

**Comment by RussoOrmon** [comment]
checked

File: src/components/feedback-slider/FeedbackSlider.jsx [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+      left: '470px',
+   },
+   '& .slick-prev': {
+      position: 'absolute',
+      top: '310px',
+      zIndex: '8',
+      left: '273px',
+   },
+   '& .slick-next:hover, .slick-prev:hover': {
+      circle: {
```
Comment (erkinovakurmanzhan1) [inline src/components/feedback-slider/FeedbackSlider.jsx:184]: checked

File: src/components/feedback-slider/FeedbackSlider.jsx [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+   return (
+      <>
+         <StyledInfo>
+            Отзывы наших <span> пациентов</span>
+         </StyledInfo>
+         <MainContainer>
+            <StyledSlider {...settings}>
+               {infoSlide.map((item) => {
+                  return (
+                     <Container>
```
Comment (songhee24) [inline src/components/feedback-slider/FeedbackSlider.jsx:83]: where is the key prop ?

File: src/components/feedback-slider/FeedbackSlider.jsx [hunk 4]
```diff
@@ -0,0 +1,209 @@
...
+         style={{ ...style }}
+      />
+   )
+}
+
+const customDots = (dots) => <div>{dots}</div>
+const customPaging = () => <Pagination />
+
+export const FeedbackSlider = () => {
+   const settings = {
```
Comment (songhee24) [inline src/components/feedback-slider/FeedbackSlider.jsx:63]: need auto play also

**Review by songhee24 — CHANGES_REQUESTED** [review]

**Review by songhee24 — APPROVED** [review]

**Pull request merged**